src/js/controller/model.js

```diff
@@ -187,6 +187,12 @@ class Model extends SimpleModel {
     // Mobile players always wait to become viewable.
     // Desktop players must have autostart set to viewable
     setAutoStart(autoStart) {
+        // Always autostart for tizen app
+        if (this.attributes.isTizenApp) {
```

Comment:
Where is `isTizenApp` being set?

Can we rename this to something more general for TV applications?

Reply:
Currently have this set in the player config in the application's UI.
https://github.com/jwplayer/jwplayer-tizen-app/blob/381d394a04a334ec0fc1cff7726e43a35f0052a2/JWTizenApp/js/tizen-player.js#L7

Can definitely rename to make more general.

Reply:
Yeah this might be too verbose, but something like  `enableSmartTvUserInterface`?